reference-architectures/sandboxes/sandbox-modules/tf/system-project/variables.tf

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "billing_account" {
+  description = "Billing account for the sandboxes"
+  type        = string
+}
+
+variable "sandboxes_folder" {
+  description = "Name of the folder that the system project and sandboxes will be created in."
+  type        = string
+}
+
+# variable "sandboxes_folder_parent" {
```

Comment:
why is this commented out?

Reply:
I originally had it creating the folder as well as the project, however, Terraform couldn't resolve all the values.  So we now require the folder to be created ahead of time, this was a hold over.... deleted :)